src/config.ts

```diff
@@ -1,7 +1,9 @@
 const config = {
     REPL_FILE_EXT: 'calva-repl',
     KEYBINDINGS_ENABLED_CONFIG_KEY: 'calva.keybindingsEnabled',
-    KEYBINDINGS_ENABLED_CONTEXT_KEY: 'calva:keybindingsEnabled'
+    KEYBINDINGS_ENABLED_CONTEXT_KEY: 'calva:keybindingsEnabled',
+    CURSOR_CONTEXT_IN_STRING: 'calva:inString',
```

Comment:
I think this could be a bit more descriptive, like `calva:cursorInString` just so we know it's about the cursor. Maybe that's obvious still, but doesn't hurt to be more explicit.